src/constants/constants.ts

```diff
@@ -220,7 +219,7 @@ export const windowsResourceClientPath = "SqlToolsResourceProviderService.exe";
 export const unixResourceClientPath = "SqlToolsResourceProviderService";
 export const microsoftPrivacyStatementUrl =
     "https://www.microsoft.com/en-us/privacy/privacystatement";
-export const sqlPlanFileExtension = ".sqlplan";
+export const sqlPlanLanguageId = "sqlplan";
```

Comment:
out of curiosity, why "language ID" instead of the old "file extension"?  Is this being used for more than just the file-picker?

Reply:
language id is being used for the custom editor registration in the package.json, and I figured we don't need two constants of almost the exact same thing- i can just add the dot in manually when needed